DataFormats/L1TParticleFlow/interface/layer1_objs.h

```diff
@@ -130,6 +172,11 @@ namespace l1ct {
     z0_t hwZ0;
     dxy_t hwDxy;
     tkquality_t hwQuality;
+    stub_t hwStubs;
+    redChi2Bin_t hwRedChi2RZ;    // 4 bits
+    redChi2Bin_t hwRedChi2RPhi;  // 4 bits
+    //FIXME: 3 bits would be enough
+    redChi2Bin_t hwRedChi2Bend;  // 4 bits
```

Comment:
Should this be fixed?

Reply:
Not now, the comment is for future reference.